src/overlays/actors/ovl_Bg_Vb_Sima/z_bg_vb_sima.h

```diff
@@ -7,8 +7,10 @@
 struct BgVbSima;
 
 typedef struct BgVbSima {
-    /* 0x0000 */ Actor actor;
-    /* 0x014C */ char unk_14C[0x30];
+    /* 0x0000 */ DynaPolyActor dyna;
+    /* 0x0164 */ char unk_164[0x10]; //apparently unused
+    /* 0x0174 */ s16 varianceTimer;
```

Comment:
this name isn't super explicit, what does it do?

Reply:
A lot of actors have a s16 that just counts up every frame, then use it to cause things to happen or move periodically. It was hard to come up with a generic name for this behavior.

Reply:
maybe just `timer` is better then, I thought "variance" refered to something specific
maybe `s16 timer; // used for various timings` ?